launcher/modManager/cmodlist.cpp

```diff
@@ -193,7 +255,11 @@ static QVariant getValue(QVariant input, QString path)
 		QString remainder = "/" + path.section('/', 2, -1);
 
 		entryName.remove(0, 1);
-		return getValue(input.toMap().value(entryName), remainder);
+		QMap<QString, QString> keyNormalize;
+		for(auto & key : input.toMap().keys())
+			keyNormalize[key.toLower()] = key;
```

Comment:
That's exact problem with Forge

Reply:
keyNormalize will contain the last key from input.toMap() matching its lowercased version so still depends on keys order. Thus I do not understand exactly how it normalizes things? Is it theoretically possible to have opposite situation to the current issue in order to fix which we will need the first value instead of the last one?

Reply:
They are used to be the same. The problem I'm trying to solve it that we have same mods with different name (Forge remote and forge locally). Theoretically it's possible to have two different mods remotely but even this situation should be covered (I don't remember the place, but somewhere with version overriding)